src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -79,4 +79,13 @@
     <EmbeddedResource Include="..\Sarif\Schemata\sarif-2.1.0-rtm.5.json" Link="sarif-2.1.0.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="appsettings.json">
+      <CopyToOutputDirectory Condition="'$(Configuration)' == 'Debug'">PreserveNewest</CopyToOutputDirectory>
```

Comment:
>Condition="'$(Configuration)' == 'Debug'" [](start = 29, length = 41)

Although I added an appsettings.json file to the Multitool project, it doesn't get packaged in the Release build. i.e. I added it here for devs (and as an example.)